modules/ossm-supported-configurations.adoc

```diff
@@ -58,6 +58,7 @@ endif::openshift-rosa[]
 == Supported configurations for Kiali
 
 * The Kiali console is only supported on the two most recent releases of the Chrome, Edge, Firefox, or Safari browsers.
+* The `openshift` authentication strategy is the only supported authentication configuration when Kiali is deployed with OpenShift Service Mesh (OSSM). The `openshift` strategy controls access based on the individual's role-based access control (RBAC) roles in OpenShift.
```

Comment:
- Should you use an attribute for `OpenShift Service Mesh`? 
- Add an attribute to replace `OpenShift`.